Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -265,14 +242,10 @@ AppDelegateViewModelOutputs {
       .filter { AppEnvironment.current.currentUser != nil }
 
     if #available(iOS 10.0, *) {
-      self.getNotificationAuthorizationStatus =
-        Signal.merge(
-          applicationLaunchedWithUser,
-          self.notificationAuthorizationCompletedProperty.signal)
+      self.getNotificationAuthorizationStatus = applicationLaunchedWithUser
 
-      self.registerForRemoteNotifications = self.notificationAuthorizationStatusProperty.signal
-        .skipNil()
-        .filter { $0.isAuthorized }
+      self.registerForRemoteNotifications = self.notificationAuthorizationCompletedProperty.signal
+        .filter(isTrue)
         .ignoreValues()
```

Comment:
We want to register if the result of `requestAuthorization()` is true, not check the status of `getNotificationSettings()`. A user with an `.authorized` status has *already* registered for remote notifications.